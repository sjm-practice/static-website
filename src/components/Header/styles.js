const styles = () => ({
  container: {
    width: "100%",
    height: "300px",
    background: "darkgreen",
  },
});

export default styles;
