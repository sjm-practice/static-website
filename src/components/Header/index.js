import React from "react";
import Typography from "@material-ui/core/Typography";
import { withStyles } from "@material-ui/core/styles";
import styles from "./styles";

const Header = ({ classes }) => {
  return (
    <div id="Header" className={classes.container}>
      <Typography variant="h5">Header</Typography>
    </div>
  );
};

export default withStyles(styles)(Header);
