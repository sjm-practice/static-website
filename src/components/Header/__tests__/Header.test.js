/* eslint-env jest */
import React from "react";
import TestRenderer from "react-test-renderer";
import Header from "../index";

describe("<Header />", () => {
  it("should render and match snapshot", () => {
    const testRenderer = TestRenderer.create(<Header />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
