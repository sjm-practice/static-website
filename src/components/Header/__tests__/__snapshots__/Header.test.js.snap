// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render and match snapshot 1`] = `
<div
  id="Header"
>
  Header
</div>
`;
